dapr/clients/grpc/client.py

```diff
@@ -282,3 +283,62 @@ def publish_event(
         return DaprResponse(
             headers=call.initial_metadata(),
             trailers=call.trailing_metadata())
+
+    def get_secret(
+            self,
+            store_name: str,
+            key: str,
+            metadata: Optional[MetadataTuple] = ()) -> GetSecretResponse:
```

Comment:
this metadata is for grpc metadata. 

For secret store metadata, please add new arg - secret_metadata: Dict[str, str]

```
   def get_secret(
            self,
            store_name: str,
            key: str,
            secret_metadata: Optional[Dict[str, str]] = {},
            metadata: Optional[MetadataTuple] = ()) -> GetSecretResponse:
```

Reply:
I was confused b/w them. Changed it.